.github/actions/setup-environment/action.yml

```diff
@@ -312,23 +317,40 @@ runs:
           echo "gp2-api-url=https://api-$PR_NUMBER.gp2.asap.science" >> $GITHUB_OUTPUT
 
           if [ "$CRN_CREATE_CONTENTFUL_PR_ENV" = true ]; then
-            echo "setting up dedicated PR env"
-            echo "crn-algolia-index=asap-hub_CI-$PR_NUMBER" >> $GITHUB_OUTPUT
-            echo "crn-analytics-algolia-index=crn-analytics_CI-$PR_NUMBER" >> $GITHUB_OUTPUT
+            echo "setting up dedicated Contentful PR env"
             echo "crn-contentful-env-id=crn-$PR_NUMBER" >> $GITHUB_OUTPUT
+
+            if [ "$CRN_CREATE_ALGOLIA_PR_ENV" = true ]; then
+              echo "setting up dedicated Algolia Analytics PR env"
```

Comment:
I think we would have something like

```
          if [ "$CRN_CREATE_CONTENTFUL_PR_ENV" = true ]; then
            echo "setting up dedicated Contentful PR env and Algolia Entity Index"
            echo "crn-algolia-index=asap-hub_CI-$PR_NUMBER" >> $GITHUB_OUTPUT           # Entity Algolia index points to a new PR index
            echo "crn-analytics-algolia-index=crn-analytics_dev" >> $GITHUB_OUTPUT      # Analytics Algolia keeps pointing to dev
            echo "crn-contentful-env-id=crn-$PR_NUMBER" >> $GITHUB_OUTPUT               # Contentful points to a new PR env
          elif [ "$CRN_CREATE_ANALYTICS_ALGOLIA_INDEX" = true ]; then
            echo "setting up Contentful and Entity Algolia Index pointing to DEV and a dedicated Analytics Algolia Index"
            echo "crn-algolia-index=asap-hub_dev" >> $GITHUB_OUTPUT                          # Entity Algolia index points to dev
            echo "crn-analytics-algolia-index=crn-analytics_CI-$PR_NUMBER" >> $GITHUB_OUTPUT # Analytics Algolia points to a new PR index
            echo "crn-contentful-env-id=Development" >> $GITHUB_OUTPUT                       # Contentful points to dev
          else
            echo "setting up PR pointing to DEV env"
            echo "crn-algolia-index=asap-hub_dev" >> $GITHUB_OUTPUT
            echo "crn-analytics-algolia-index=crn-analytics_dev" >> $GITHUB_OUTPUT
            echo "crn-contentful-env-id=Development" >> $GITHUB_OUTPUT
          fi
```

Because the new label `crn-create-analytics-algolia-index` would be responsible to create only an Analytics Algolia index while Contentful points do Development and Entity Algolia Index points to `asap-hub_dev`